cmp/cmpopts/ignore.go

```diff
@@ -112,6 +112,10 @@ func (tf ifaceFilter) filter(p cmp.Path) bool {
 // In particular, unexported fields within the struct's exported fields
 // of struct types, including anonymous fields, will not be ignored unless the
 // type of the field itself is also passed to IgnoreUnexported.
+//
+// Avoid ignoring unexported fields of a type which you do not control (i.e. a
+// type from another package), as changes to the implementation of such types
```

Comment:
s/package/repository/g

Repositories are generally the right boundary for ownership, not individual Go packages.

Reply:
done